.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+      - id: black
+        language_version: python3.9
```

Comment:
what does python3.9 mean here? Can it have conflicts with people using other versions?

Reply:
It was on the default template. As far as I understood, it determines the python version for the hook. Black is called with this version. I assume it should be compatible with other versions such as 3.7...3-10.

Reply:
Let's merge it then and have a bugfix if it does not work.